### PR TITLE
Added flag to keep output.

### PR DESCRIPTION
**Pull request by yunikkk**

### CODE REVIEW

**Comment by trevjonez** [comment]
are the current output file names meaningful to humans? IIRC it was a random number or some such? It be helpful to make it something deterministic in this same PR as I would like to implement this option in the gradle plugin with the ability to move the output files into the variant directory adjacent to the report outputs. Currently they pop up in the consuming module root dir due to the way gradle works.

**Comment by yunikkk** [comment]
Currently only `instrumentation.output` file is named meaningfully, other logs from adb commands have random names (and stored in the root dir). To change that we'll need to modify another library, `commander`. I'm going to do it, but seems it'll better be the in the next PR.

**Comment by artem-zinnatullin** [comment]
@trevjonez see https://github.com/gojuno/commander/pull/6#discussion_r143263466

It's definitely possible

**Comment by yunikkk** [comment]
@artem-zinnatullin PTAL)

File: composer/src/main/kotlin/com/gojuno/composer/Args.kt [hunk 1]
```diff
@@ -85,11 +86,19 @@ private class JCommanderArgs {
             names = arrayOf("--verbose-output"),
             required = false,
             arity = 1,
-            description = "Either `true` or `false` to enable/disable verbose output for Swarmer. `false` by default."
```
Comment (artem-zinnatullin) [inline composer/src/main/kotlin/com/gojuno/composer/Args.kt]: LOL

Comment (artem-zinnatullin) [reply composer/src/main/kotlin/com/gojuno/composer/Args.kt]: I guess it's my copypasta, don't want to look into history :)

**Pull request merged**